packages/react-cosmos-webpack/src/__tests__/entry.js

```diff
@@ -43,7 +43,7 @@ describe('loader path', () => {
   let options;
 
   beforeAll(() => {
-    init('/loader/');
+    init('./loader/index.html');
```

Comment:
This value mocks `window.location.pathname`, so it should be `/loader/index.html`. Tests should continue to pass.